Expensify/App
# Update Onfido SDK styles to match our style/brand

### Details
Customizes the Onfido modal UI to look more in line with our brand style.

cc @shawnborton 

### Fixed Issues
$ https://github.com/Expensify/Expensify/issues/179228

### Tests
1. Log into an account with a Workspace but without a verified bank account.
2. Navigate to `Settings > Select Workspace > Expensify Card > Get started`
3. Use the information in [this SO](https://stackoverflow.com/c/expensify/questions/342) to add a bank account in the `VERIFYING` state.
4. When you reach the Onfido step, verify that the style looks like the screenshots below:

### QA Steps
Steps above.

### Tested On


### Screenshots


#### Web
![Screen Shot 2021-09-29 at 1 10 18 PM](https://user-images.githubusercontent.com/22219519/135333416-02545417-d57e-48b1-bcdf-61f26bff38b9.png)
![Screen Shot 2021-09-29 at 1 10 26 PM](https://user-images.githubusercontent.com/22219519/135333417-1cb72281-7041-4bc8-94dc-ca8ab6834b12.png)
![Screen Shot 2021-09-29 at 1 10 33 PM](https://user-images.githubusercontent.com/22219519/135333419-a7d926ad-7173-48c5-b6fc-f753c7f0cc25.png)
![Screen Shot 2021-09-29 at 1 10 55 PM](https://user-images.githubusercontent.com/22219519/135333422-5917d21d-8bf5-4ff5-9222-c64386ec392c.png)

#### Mobile Web

![Simulator Screen Shot - iPhone 12 - 2021-09-29 at 13 18 40](https://user-images.githubusercontent.com/22219519/135334442-7e30df18-b7f0-4d63-a9c1-9b8556d5abce.png)
![Simulator Screen Shot - iPhone 12 - 2021-09-29 at 13 18 43](https://user-images.githubusercontent.com/22219519/135334446-6e02f1f1-5c22-4513-84b3-d3598afaba4d.png)
![Simulator Screen Shot - iPhone 12 - 2021-09-29 at 14 34 47](https://user-images.githubusercontent.com/22219519/135344479-46431e8c-93ba-48c7-9615-f11fbbdefb54.png)


#### Desktop

![Screen Shot 2021-09-29 at 1 13 39 PM](https://user-images.githubusercontent.com/22219519/135333983-a1c87766-05b7-492f-b96a-96a4ecdfcd18.png)
![Screen Shot 2021-09-29 at 1 13 46 PM](https://user-images.githubusercontent.com/22219519/135333987-f97c3974-9f2b-430a-93bc-0a7a686f5812.png)
![Screen Shot 2021-09-29 at 1 13 53 PM](https://user-images.githubusercontent.com/22219519/135333989-fcd488d7-1a85-489b-9855-adf601016cba.png)
![Screen Shot 2021-09-29 at 1 14 09 PM](https://user-images.githubusercontent.com/22219519/135333992-cf82ace1-724f-4f2a-b44f-47432cfa78a4.png)

#### iOS

![Simulator Screen Shot - iPhone 12 - 2021-09-29 at 14 25 00](https://user-images.githubusercontent.com/22219519/135343200-252e8d0b-5b56-4233-87c7-f5dd07d82b48.png)

#### Android

![Screenshot_1632945745](https://user-images.githubusercontent.com/22219519/135343210-0858c0a9-cdb0-428e-b7d7-c037571968b1.png)
